drs/drs-assembly/subscribe-model.conf

```diff
@@ -749,18 +529,10 @@ Information about current dither pattern position. Contains:
 
 {
   subsystem =  NFIRAOS
-  component =  nfiraos-tbd
```

Comment:
I could only find a single sodium layer value published (i.e., not one per each of the 6 x LGS). But perhaps this information is provided elsewhere.